hierarchical_planner:
  # Defining the goal state
  goal: {x: 3.5, y: 0, yaw: 0}
  
  # Defining the body planner
  body_planner:
    # There are several solver such as AnytimeRepairingAStar, AStar and Dijkstrap
    # The default solver is the ARA*
    path_solver: AnytimeRepairingAStar
    ###### TODO: pose_solver:
    # There are several adjacency model such as LatticeBasedBodyAdjacency and GridBasedBodyAdjacency
    # The default adjacency model is LatticeBasedBody without body features
    adjacency:
      name: LatticeBasedBodyAdjacency
      features:
        potential_leg_collision: {enable: false, weight: 500}
        potential_body_orientation: {enable: true, weight: 500}
    # The maximum allowed computation time for path solver in seconds. Note that this time is used only in Anytime algorithms
    # The default computation time is 1 sec.
    path_computation_time: 5
    ###### TODO: pose_computation_time:
    
  contact_planner:
    solver: GreedyAlgorithm ###### TODO
    features:
      maximum_height: {enable: false, weight: 5}
      body_orientation: {enable: false, weight: 5}
    # The maximum allowed computation time for contact solver in seconds
    # The default computation time is 1 sec.
    computation_time: 0.5
    